File: problems/prob10/prob10.py
from random import randrange

def rand_sum(num):
	total = 0
	for x in str(num):
		total += int(x)
	return total

rand_num = randrange(1000,9999)
print "The number is " + str(rand_num)
print "The sum of its digits is " + str(rand_sum(rand_num))

File: problems/prob10/prob10.rb
def rand_sum(num)
	total = 0
	(num.to_s).each_char do |x|
		total += (x.to_i)
	end
	return total
end

rand_num = rand(1000..9999)
puts "The number is " + rand_num.to_s + "\n"
puts "The sum of its digits is " + (rand_sum(rand_num)).to_s
